### PR TITLE
Fixing small issues when running staking testnet

### PR DESCRIPTION
**Pull request by ilblackdragon**
- Use consensus params from config for real.
 - Added override flags for port, rpc port, boot nodes and min peers for easier configuring local testnets.
 - Decreased epoch length to 60 blocks, to allow people to quickly become validators.
 - Fixed an issue with rotation of validators.

Example of running sample testnet within local environment:

CMD 1:
```
./target/debug/near init
./target/debug/near run
```

CMD 2:
```
./target/debug/near --home=<some/folder> init --account-id=illia.near
cp ~/.near/genesis.json <some/folder>/genesis.json
./target/debug/near --home=<some/folder> run --boot-nodes=<peer id of node above>@127.0.0.1:24567 --network-addr=0.0.0.0:24568 --rpc-addr=0.0.0.0:3031
```

CMD 3:
```
# To get public key for staking:
cat <some/folder>/validator_key.json

near-shell create_account illia.near
near-shell stake illia.near <staking public key from above> <amount to stake>
```

Wait until the CMD 2 becomes a staking node (you will see it based on 4th column becoming `V/2` - meaning this node is Validator and total number is 2.

### CODE REVIEW

File: chain/network/src/peer_manager.rs [hunk 1]
```diff
@@ -281,7 +281,7 @@ impl Actor for PeerManagerActor {
         if let Some(server_addr) = self.config.addr {
             // TODO: for now crashes if server didn't start.
             let listener = TcpListener::bind(&server_addr).unwrap();
-            info!(target: "network", "Server listening at {}@{}", self.peer_id, server_addr);
+            info!(target: "info", "Server listening at {}@{}", self.peer_id, server_addr);
```
Comment (bowenwang1996) [inline chain/network/src/peer_manager.rs]: Why is target `info`?

File: near/src/config.rs [hunk 2]
```diff
@@ -446,9 +456,13 @@ pub fn init_configs(
             // Create new configuration, key files and genesis for one validator.
             let mut config = Config::default();
             config.network.skip_sync_wait = true;
+            if fast {
+                config.consensus.min_block_production_delay = Duration::from_millis(10);
```
Comment (bowenwang1996) [inline near/src/config.rs]: Should this be extracted out as a constant?

File: near/src/runtime.rs [hunk 3]
```diff
@@ -78,6 +78,21 @@ impl NightshadeRuntime {
...
+        let validator_assignemnt = vm.get_validators(epoch)?;
+        let total_seats: u64 = validator_assignemnt.block_producers.iter().sum();
+        let mut cur_seats = 0;
+        for (i, seats) in validator_assignemnt.block_producers.iter().enumerate() {
+            if cur_seats + seats > idx % total_seats {
+                return Ok(validator_assignemnt.validators[i].clone());
+            }
+            cur_seats += seats;
+        }
+        unreachable!()
```
Comment (bowenwang1996) [inline near/src/runtime.rs]: Shouldn't this be some error instead of `unreachable!()`?

File: chain/network/src/peer_manager.rs (hunk 1 above)
Comment (ilblackdragon) [reply chain/network/src/peer_manager.rs]: To show it at non verbose mode.

File: near/src/runtime.rs (hunk 3 above)
Comment (ilblackdragon) [reply near/src/runtime.rs]: That's the error - it should never reach it otherwise there is a bug in math.

Comment (bowenwang1996) [reply near/src/runtime.rs]: Okay. If that's the case why do you need to return `Result`

File: chain/network/src/peer_manager.rs (hunk 1 above)
Comment (bowenwang1996) [reply chain/network/src/peer_manager.rs]: I see. That feels weird. I think we shouldn't filter out `info` for any target

**Pull request merged**